### PR TITLE
Mark docrep to remote migration as GA and modify settings names

### PR DESCRIPTION
**Pull request by gbbafna**
### Description
Mark docrep to remote migration as GA and modify settings names

### Issues Resolved
Resolves https://github.com/opensearch-project/documentation-website/issues/7343 

### Version

OS 2.15 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md [hunk 1]
```diff
@@ -105,14 +102,14 @@ As of OpenSearch 2.14, OpenSearch nodes cannot be migrated back to document repl
    }
    ```
 
-1. Set the `remote_store.compatibility_mode` setting to `mixed` to allow remote-backed storage nodes to join the cluster. Then set `migration.direction` to `remote_store`, which allocates new indexes to remote-backed data nodes. The following example updates the aforementioned setting using the Cluster Settings API:
+1. Set the `cluster.remote_store.compatibility_mode` setting to `mixed` to allow remote-backed storage nodes to join the cluster. Then set `migration.direction` to `remote_store`, which allocates new indexes to remote-backed data nodes. The following example updates the aforementioned setting using the Cluster Settings API:
```
Comment (BhumikaSaini-Amazon) [inline _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:105]:
```suggestion
1. Set the `cluster.remote_store.compatibility_mode` setting to `mixed` to allow remote-backed storage nodes to join the cluster. Then set `cluster.migration.direction` to `remote_store`, which allocates new indexes to remote-backed data nodes. The following example updates the aforementioned setting using the Cluster Settings API:
```

**Review by BhumikaSaini-Amazon — CHANGES_REQUESTED** [review]

**Comment by hdhalter** [comment]
Thanks, @gbbafna! Is this ready for review by the doc team?

**Comment by gbbafna** [comment]
> Thanks, @gbbafna! Is this ready for review by the doc team?

Yes, it is .

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md [hunk 2]
```diff
@@ -24,9 +21,9 @@ Rolling upgrades, sometimes referred to as *node replacement upgrades*, can be p
 
 Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
 
-Before migrating to remote-backed storage, upgrade to OpenSearch 2.14.
+Before migrating to remote-backed storage, upgrade to OpenSearch 2.15.
```
Comment (vagimeli) [inline _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:24]:
```suggestion
Before migrating to remote-backed storage, upgrade to the latest version of OpenSearch.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md [hunk 3]
```diff
@@ -8,9 +8,6 @@ grand_parent: Availability and recovery
 
 # Migrating to remote-backed storage
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/7986).    
-{: .warning}
-
 Introduced 2.14
```
Comment (vagimeli) [inline _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:11]: This label is confusing. Is this no longer an experimental feature? If so, then we should update label to Introduced 2.15. The subsequent text refers to 2.15.

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md [hunk 4]
```diff
@@ -24,9 +21,9 @@ Rolling upgrades, sometimes referred to as *node replacement upgrades*, can be p
 
 Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
 
-Before migrating to remote-backed storage, upgrade to OpenSearch 2.14.
+Before migrating to remote-backed storage, upgrade to OpenSearch 2.15.
 
-As of OpenSearch 2.14, OpenSearch nodes cannot be migrated back to document replication. If you need to revert the migration, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
+As of OpenSearch 2.15, OpenSearch nodes cannot be migrated back to document replication. If you need to revert the migration, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
```
Comment (vagimeli) [inline _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:26]:
```suggestion
As of OpenSearch 2.15, nodes cannot be migrated back to document replication. To revert, perform a fresh installation and restore from a snapshot. Take a snapshot and store it remotely before upgrading.
```

**Review by vagimeli — COMMENTED** [review]
@gbbafna Please see my doc review comment. Once you respond, I'll make necessary changes and then move this to editorial. Thank you.

**Review by BhumikaSaini-Amazon — APPROVED** [review]

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md (hunk 3 above)
Comment (gbbafna) [reply _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:11]: Yes, we are doing GA for it .

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md (hunk 2 above)
Comment (gbbafna) [reply _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:24]: They need to minimally upgrade to 2.15 for migration , latest versions is not mandatory .

File: _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md (hunk 4 above)
Comment (gbbafna) [reply _tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md:26]: `Storing it remotely` doesn't spell out need to do a snapshot .

**Review by vagimeli — APPROVED** [review]
Doc review complete. Addressed SME comments. Moving to editorial.

**Pull request merged**